project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "root",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "./",
+  "targets": {
+    "test:format": { "command": "pnpm run prettier --check" },
+    "build": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "rollup --config rollup.config.js",
+          "nx run-many --target=build:types",
+          "nx build @tanstack/eslint-plugin-query",
+          "nx build @tanstack/svelte-query"
+        ],
+        "parallel": true
+      }
+    },
```

Comment:
Build should do these 4 things, and it can actually do all 4 in independently if we want, so that's what we're doing here.

Interestingly `@tanstack/svelte-query` has it's own build step, but all other packages are built in 1 go with the `rollup` command it appears.

@TkDodo - let me know if there's a preference here, we might be able to get some boost by parallelizing the package builds and running them individually (this also allows the cache to be more granular - as it stands we need to "invalidate" the build cache for all projects whenever a single project's public code is touched 🥶

If we do want to break it up, let's do it in another PR! As I mention elsewhere, I'd like to keep this one as "like-for-like" as possible!